specification/mediaservices/resource-manager/readme.md

```diff
@@ -39,6 +39,20 @@ input-file:
 - Microsoft.Media/stable/2015-10-01/media.json
 ```
 
+### Tag: package-2018-03-preview

```

Comment:
Please add this tag to the build config for Go (I wasn't able to push to your branch), specifically add a line to the ```Go multi-api``` section and make a copy of ```package-2015-10 and go``` for this API version.

Reply:
Fixed.